jenkins/ci.suse.de/pipelines/socok8s-airship-from-rpm.Jenkinsfile

```diff
@@ -0,0 +1,181 @@
+            steps {
+                script {
+                    in_container('printenv')
+                }
+            }
+        }
+
+        stage('Copy in cloud config and sshkey') {
+            steps {
+                script {
+                    in_container('mkdir -p /root/.ssh')
+                    sh 'podman cp ~/.ssh/id_rsa $CONTAINER_ID:/root/.ssh/id_rsa'
+                    sh 'podman cp ~/.ssh/id_rsa.pub $CONTAINER_ID:/root/.ssh/id_rsa.pub'
+                    in_container('touch /root/.ssh/known_hosts')
+                    in_container('mkdir -p /etc/openstack')
+                    sh 'podman cp ~/.config/openstack/clouds.yaml $CONTAINER_ID:/etc/openstack/clouds.yaml'
+                }
+            }
+        }
+
+        stage('Create network') {
+            options {
+                timeout(time: 10, unit: 'MINUTES', activity: true)
+            }
+            steps {
+                socok8s_run("deploy_network")
+            }
+        }
+
+        stage('Create VMs') {
+            options {
+                timeout(time: 45, unit: 'MINUTES', activity: true)
+            }
+            parallel {
+                stage('Deploy CaaSP') {
+                    steps {
+                        socok8s_run("deploy_caasp")
+                    }
+                }
+                stage('Deploy SES') {
+                    steps {
+                        socok8s_run("deploy_ses")
+                    }
+                }
+                stage('Deploy CCP Deployer') {
+                    steps {
+                        socok8s_run("deploy_ccp_deployer")
+                    }
+                }
+            }
+        }
+
+        stage('Configure CaaSP workers') {
+            options {
+                timeout(time: 10, unit: 'MINUTES', activity: true)
+            }
+            steps {
+                socok8s_run("enroll_caasp_workers")
+                socok8s_run("setup_caasp_workers_for_openstack")
+            }
+        }
+
+        stage('Deploy Airship') {
+            options {
+                timeout(time: 45, unit: 'MINUTES', activity: true)
+            }
+            steps {
+                socok8s_run("setup_airship")
+            }
+        }
+    }
+
+    post {
+        failure {
+            script {
+                if (env.hold_instance_for_debug == 'true') {
+                    echo "You can reach this node by connecting to its floating IP as root user, with the default password of your image."
+                    timeout(time: 3, unit: 'HOURS') {
+                        input(message: "Waiting for input before deleting env ${SOCOK8S_ENVNAME}.")
+                    }
+                }
+            }
+            script {
+                socok8s_run('gather_logs')
+            }
+            zip archive: true, dir: 'logs/', zipFile: 'logs.zip'
+            archiveArtifacts artifacts: 'logs.zip'
+        }
+        cleanup {
```

Comment:
should we bring the gather_logs part from the PR job in here as well?

Reply:
Sorry, could you please link me to which part you mean?

Reply:
https://github.com/SUSE-Cloud/socok8s/blob/master/Jenkinsfile.integration#L189

Reply:
this can come later thougth

Reply:
This is already there in the failure part just above.